plugin.yaml

```diff
@@ -879,6 +879,40 @@ node_types:
             openstack_config:
               default: {}
 
+  cloudify.openstack.nodes.Stack:
+    derived_from: cloudify.nodes.Root
+    properties:
```

Comment:
add property `stack` which can be accept a similar request content: https://developer.openstack.org/api-ref/orchestration/v1/index.html#stacks.

Reply:
As I understand, property "stack" must contains data, like in this example:
https://developer.openstack.org/api-ref/orchestration/v1/index.html#request-example

Plugin must put value of this property to request body.

Reply:
We need to use the "stack" property. This is consistent with the design of other node templates. (The plugin YAML says this pattern is deprecated, but it is not. We want to use this pattern. The inputs "args" will override "stack" as is consistent with all other types.